en/lessons/advanced/gen-stage.md

```diff
@@ -209,9 +209,9 @@ def start(_type, _args) do
   import Supervisor.Spec, warn: false
 
   children = [
-    worker(GenstageExample.Producer, [0]),
-    worker(GenstageExample.ProducerConsumer, []),
-    worker(GenstageExample.Consumer, [])
+    {GenstageExample.Producer, 0},
+    {GenstageExample.ProducerConsumer, []},
```

Comment:
I've opted for the `{Module, []}` style syntax here instead of just passing the module directly for consistency later on when talking about multiple consumers